ts/Stock/Indicators/DisparityIndex/DisparityIndexOptions.d.ts

```diff
@@ -0,0 +1,34 @@
+/* *
+ *
+ *  License: www.highcharts.com/license
+ *
+ *  !!!!!!! SOURCE GETS TRANSPILED BY TYPESCRIPT. EDIT TS FILE ONLY. !!!!!!!
+ *
+ * */
+
+/* *
+ *
+ *  Imports
+ *
+ * */
+
+import type {
+    SMAOptions,
+    SMAParamsOptions
+} from '../SMA/SMAOptions';
+
+/* *
+ *
+ *  Declarations
+ *
+ * */
+
+export interface DisparityIndexOptions extends SMAOptions {
+    params?: DisparityIndexParamsOptions;
+}
+
+export interface DisparityIndexParamsOptions extends SMAParamsOptions {
+    average: 'sma'|'ema'|'dema'|'tema'|'wma';
```

Comment:
Shouldn't we place there only the type of the `average: string`?

Reply:
I think that both are fine, however, `'sma'|'ema'|'dema'|'tema'|'wma'` type is more accurate.

Reply:
I believe `someRandomString` is an invalid type of `average` 🤔